### PR TITLE
Integrate Dublin proofing changes to Deployment Guide

### PR DESCRIPTION
**Pull request by lproven**
### Description
Integrate all the changes requested by Dublin proofreading of the DepGuide.

One is possibly controversial: we are not sure if "DHCPv6 DHCP Unique Identifier" is necessary. It seems tautological. As requested, I have removed the 2nd occurrence to leave it as "DHCPv6 Unique Identifier". If the longer version is required because it is formally correct, it is trivial to reverse.

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLE15SP1
- [ ] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

File: xml/deployment_image.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
    up and clones can be used immediately.
   </para>
   <para>
-   The <command>clone-master-clean-up</command> command for example removes:
```
Comment (taroth21) [inline xml/deployment_image.xml:59]: To prevent interrupting the train of thought, I would move 'For example,' to the beginning of the sentence.

**Comment by lproven** [comment]
> one file (`deployment_prep_power.xml`) has a very large diff, could you please check again if you accidentally reformatted it completely while integrating the proofing corrections?

That was intentional -- I think it got reflowed automatically in the past and the indentation was a mess, so I did it with ``daps-xmlformat``.

**Comment by taroth21** [comment]
> > one file (`deployment_prep_power.xml`) has a very large diff, could you please check again if you accidentally reformatted it completely while integrating the proofing corrections?
> 
> That was intentional -- I think it got reflowed automatically in the past and the indentation was a mess, so I did it with `daps-xmlformat`.

I see - but for the future, please use a separate commit (outside of the PR) for reformatting a complete file (as already suggested by Stefan in another case). Otherwise it's hard to identify the proofing changes as such and impossible to cherry-pick any proofing commits, if required.

**Comment by lproven** [comment]
> I see - but for the future, please use a separate commit (outside of the PR) for reformatting a complete file

Oh. OK then.

> (as already suggested by Stefan in another case). Otherwise it's hard to identify the proofing changes as such and impossible to cherry-pick any proofing commits, if required.

Fair point!

**Pull request merged**